tirocks-sys/build.rs

```diff
@@ -258,5 +264,11 @@ fn main() {
         build.flag("-fno-rtti");
     }
     link_cpp(&mut build);
-    build.warnings(false).compile("libcrocksdb.a");
+    // It can break ABI if the return type is too small. But we only return
```

Comment:
Why exactly? Do you have a referencing document I can learn about?

Reply:
Turns out I'm wrong, struct size larger than 8 bytes can still be returned in registers. So this PR will not work.